src/lib/features/export-import-toggles/export-import-service.ts

```diff
@@ -362,24 +362,23 @@ export default class ExportImportService {
         const unsupportedContextFields = await this.getUnsupportedContextFields(
             dto,
         );
-        if (
-            Array.isArray(unsupportedContextFields) &&
-            unsupportedContextFields.length > 0
-        ) {
-            throw new UnleashError({
-                name: 'BadDataError',
-                message:
-                    'Some of the context fields you are trying to import are not supported.',
-                // @ts-ignore-error We know that the array contains at least one
-                // element here.
-                details: unsupportedContextFields.map((field) => {
+        if (Array.isArray(unsupportedContextFields)) {
+            const [firstError, ...remainingErrors] =
+                unsupportedContextFields.map((field) => {
                     const description = `${field.name} is not supported.`;
                     return {
                         description,
                         message: description,
                     };
-                }),
-            });
+                });
+            if (firstError) {
```

Comment:
Ensure that `firstError` actually exists. The compiler is pretty smart, but not infallible. For instance, if you do:

```ts
// create an empty list
const xs: { name: string }[] = []

// map and split it
const [head, ...tail] = xs

// try and assign it to a non-empty list
details: [head, ...tail]
```
It will gladly accept it. 

So instead, we need to verify that `firstError` actually **does** exist.